service/apis/entity_service.go

```diff
@@ -37,12 +37,11 @@ type IEntityService interface {
 // EntityService provides service for controller and frontend by implement IEntityService interface
 type EntityService struct {
 	dbclient db.IDGClient
-	metaSvc  *MetaService
 }
 
 // NewEntityService creates a new EntityService with the given dgraph client.
-func NewEntityService(dc db.IDGClient, msvc *MetaService) *EntityService {
-	return &EntityService{dc, msvc}
+func NewEntityService(dc db.IDGClient) *EntityService {
+	return &EntityService{dc}
 }
 
```

Comment:
Good to know that NewEntityService only takes dbClient as input.

Reply:
Had to update all related tests